docs/Aedes.md

```diff
@@ -386,6 +387,28 @@ aedes.authorizeForward = function (client, packet) {
 }
 ```
 
+## Handler: preUnsubscribe (client, packet)
+
+- client: [`<Client>`](./Client.md)
+- packet: `<aedes-packet>` & [`UNSUBSCRIBE`][UNSUBSCRIBE]
+
+Invoked then a client unsubscribes from topics. Just like authorizeSubscribe, this function allows modifying the topics before executing the unsubscribe. This may be needed in cases where topics were modified in authorizeSubscribe. For example:
```

Comment:
```suggestion
Invoked when a client unsubscribes from topics. Just like `authorizeSubscribe`, this function allows modifying the topics before executing the unsubscribe. This may be needed in cases where topics were modified in authorizeSubscribe. For example:
```